### ISSUE TITLE
Only GET actions?

### ISSUE THREAD

**Issue by WhatFreshHellIsThis** [nbarbettini/BeautifulRestApi#10]
Am I missing something?  There seem to only be GET routes, no post or put or patch examples?

**Comment by nbarbettini** [comment]
You're right, there should be some POST/PUT examples. This repo isn't quite as feature-complete as the code in my video course (not intentionally, just for a lack of time). Sorry about that.

I'll leave this issue open to remind myself to come back to this. Thanks!
